src/index.js

```diff
@@ -180,6 +182,8 @@ class ReactRelayQueryRenderer extends React.Component<Props, State> {
                         return;
                     }
                     snapshot = environment.lookup(operation.fragment);
+                    this._rootSubscription = environment.subscribe(snapshot, this._onChange);
```

Comment:
Is this needed here? the same line is also a few lines down.

Reply:
Good catch, updated